comp/core/autodiscovery/autodiscoveryimpl/autoconfig.go

```diff
@@ -8,12 +8,18 @@ package autodiscoveryimpl
 
 import (
 	"context"
+	"encoding/json"
+	"net/http"
+	"sort"
 	"sync"
 	"time"
 
 	"go.uber.org/atomic"
 	"go.uber.org/fx"
 
+	"github.com/DataDog/datadog-agent/comp/api/api"
+	"github.com/DataDog/datadog-agent/comp/api/api/apiimpl/response"
+	"github.com/DataDog/datadog-agent/comp/api/api/apiimpl/utils"
```

Comment:
Same comment as the one above. The utils should be part of the  "github.com/DataDog/datadog-agent/comp/api/api" not the implementation